truss/contexts/image_builder/util.py

```diff
@@ -10,7 +10,7 @@
 # [IMPORTANT] Make sure all images for this version are published to dockerhub
 # before change to this value lands. This value is used to look for base images
 # when building docker image for a truss.
-TRUSS_BASE_IMAGE_VERSION_TAG = "v0.2.0"
+TRUSS_BASE_IMAGE_VERSION_TAG = "v0.2.1"
```

Comment:
This seems like a big change. Would be good to update pyproject version and have this match that, that way base image version will match the code version. We'll need to be careful to publish those images to dockerhub before or immediately after publishing the library to pypi. I think the pypi push is automatic, runs after integration tests pass, whole thing takes around 20 mins. I guess you could push to dockerhub right after merging to main. Please note that there's an option `--push` on `bin/generate_base_images.py` to push to dockerhub.